lib/Serialization/ModuleFormat.h

```diff
@@ -638,6 +638,19 @@ enum BlockID {
   ///
   /// \sa comment_block
   COMMENT_BLOCK_ID,
+
+  /// The module source location container block, which contains all other
+  /// source location blocks.
+  ///
+  /// This is part of a stable format and must not be renumbered!
+  MODULE_SOURCEINFO_BLOCK_ID = 192,
```

Comment:
I thought you decided these *weren't* going to be considered a stable format, since they aren't installed? As such I'm not sure you need to put them after the moduledoc numbers.

Reply:
If they are a stable format I'm a little more concerned about the format of these hash tables.

Reply:
We talked about this in person. The plan is to attempt stability, but the consequences aren't so bad if we fail.